### DISCUSSION TITLE
GitHub accelerator

### DISCUSSION THREAD

**Discussion by T-Dynamos** [termux/termux-packages#13923] [General]
https://accelerator.github.com/
Any active maintainer of termux should enroll

**Comment by agnostic-apollo** [comment]
Saw this myself too. I have made a comment about it at https://matrix.to/#/!rrHhKvLYtBVDOISphu:gitter.im/$HHLECKPu9-3ycBsXt6yz6MUGtigtm9CDQbv_VjIwPv0 if some other termux member is interested in applying as a team.

**Comment by T-Dynamos** [comment]
Today's last day 🥲
